### PR TITLE
[2.29.x] Forward ports from 2.26 (miscellaneous) 

### PR DESCRIPTION
**Pull request by malmgrens4**
(cherry picked from commit d2cb53dcb7863057a99a8654ee5b644710cee4c9)

"DDF-6360 Remove reference to deleted libs-bundles-opensaml artifact https://github.com/codice/ddf/commit/2f2d2b9aa81f5e20995810925d71215f392c6659
https://github.com/codice/ddf/pull/6403

"Fix email client not finding IMAPProvider (]https://github.com/codice/ddf/commit/c380ef9734342e567b631650d0d17d02f299ad96)
https://github.com/codice/ddf/pull/6427

"commenting out docker project for release"https://github.com/codice/ddf/commit/85a740c6fd77d636571e5329c0a553e14be9cb3d)

"Add missing features to the distribution https://github.com/codice/ddf/commit/5ea99d2c7f5d9aa5885331345e765c9462c0da69)
https://github.com/codice/ddf/pull/6431

"Fix printErrorMessage when executeWithSubject fails in SubjectCommands https://github.com/codice/ddf/commit/549643a247802829d76a84a6c445b9f1d29f21a5)
https://github.com/codice/ddf/pull/6461

[2.26.x] Match on any OR for TagsFilterDelegate
https://github.com/codice/ddf/commit/530d599c7c87cc4afbba0c4defb7440b70ef0017
https://github.com/codice/ddf/pull/6481

"[2.26.x] Updates SortedQueryMonitor to include a case insensitive sort
https://github.com/codice/ddf/commit/8d06ca4a45f0f8d78be81afbada52897cac4bda4
https://github.com/codice/ddf/pull/6500

"Copy all original source response properties to a new property https://github.com/codice/ddf/commit/5749ebf555963db54704284fc977fe2e8c8f283b)
https://github.com/codice/ddf/pull/6560

"[2.26.x] DDF-6565 create a video thumbnail service
https://github.com/codice/ddf/commit/1e89bd32c347ca6bb62c1f02296a5cef37fe9a11
https://github.com/codice/ddf/pull/6567

"Improve cql examples for catalog:remove and catalog:search https://github.com/codice/ddf/commit/38373b21a666a76cdef6c3428a01099cc9bf74fe)
https://github.com/codice/ddf/pull/6733

"Pass operation properties to populateMetacards https://github.com/codice/ddf/commit/f81ec1f3294b27c6e82f66e57d7fb793d83485ec)
https://github.com/codice/ddf/pull/6759

 Add default sort-by for ResultIterable https://github.com/codice/ddf/commit/79dbd50a692ae49fadc07f98f9524b5d3332529b
https://github.com/codice/ddf/pull/6792

"Ensure SortedQueryMonitor uses query ThreadContext
https://github.com/codice/ddf/commit/794c4e3fa58fb3fc68abeac8150e7100cc7a39cc
https://github.com/codice/ddf/pull/6784

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by malmgrens4** [comment]
build now

**Review by derekwilhelm — APPROVED** [review]

**Comment by malmgrens4** [comment]
build now

File: distribution/pom.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <module>docs</module>
         <module>ddf-catalog</module>
         <module>ddf</module>
-        <module>docker</module>
+        <!--module>docker</module-->
```
Comment (jlcsmith) [inline distribution/pom.xml:37]: we should be able to remove this change.  This was to get around a temporary build issue.

**Review by jlcsmith — APPROVED** [review]

**Comment by malmgrens4** [comment]
build now

**Review by glenhein — APPROVED** [review]

**Pull request merged**